addons/html_builder/static/src/builder/options/alignment_option.js

```diff
@@ -10,6 +11,26 @@ class AlignmentOptionPlugin extends Plugin {
                 selector: ".s_share, .s_text_highlight, .s_social_media",
             },
         ],
+        builder_actions: this.getActions(),
     };
+
+    getActions() {
+        const classAction = coreBuilderActions.classAction;
+        return {
+            setVerticalAlignment: {
+                ...classAction,
+                getPriority: ({ param: classNames = "" }) =>
+                    classNames === "align-items-stretch" ? 0 : 1,
+                isApplied: ({ editingElement, param: classNames }) => {
+                    if (classNames === "align-items-stretch") {
+                        return true;
+                    }
+                    return classNames
```

Comment:
Maybe we can directly use `classAction.isApplied()` after the `if (classNames === "align-items-stretch")`?